specification.md

```diff
@@ -1225,7 +1225,7 @@ function. They can include either static values or reference the states data inp
 | --- | --- | --- | --- |
 | [triggerEventRef](#Event-Definition) | Reference to the unique name of a `produced` event definition | string | yes |
 | [resultEventRef](#Event-Definitions) | Reference to the unique name of a `consumed` event definition | string | yes |
-| data | If string type, an expression which selects parts of the states data output to become the data (payload) of the produced event. If object type, a custom object to become the data (payload) of trigger/produced event. | string or object | no |
+| data | If string type, an expression which selects parts of the states data output to become the data (payload) of the event referenced by `triggerEventRef`. If object type, a custom object to become the data (payload) of trigger/produced event. | string or object | no |
```

Comment:
Please also update the "description" param  in the json schemas for this parameter to match the one in the specification document.

Reply:
I tried grepping the schema directory for the old description, but it returned no results:

```
╰─ grep 'If string type, an expression which selects parts of the states data output to become the data (payload) of the produced event' -ir schema/* -c
schema/common.json:0
schema/events.json:0
schema/extensions/kpi.json:0
schema/functions.json:0
schema/workflow.json:0
```
Turns out the description was not the same on the schema, which is why I missed it at first:

```
 379           "description": "If String, expression which selects parts of the states data output to become the data of the produced event. If object a custom object to become the data of produced event."
```

Will update this in a bit, thanks! :-)